src/dbc.cpp

```diff
@@ -133,6 +250,7 @@ void DbcParser::parse_dbc_messages(const std::vector<std::string>& lines) {
 			Message msg(id, name, size, node);
 
 			messages.push_back(msg);
+			continue;

```

Comment:
why do we continue here?

Reply:
If the regex matches a message is found and therefore it is not needed to check the others, because they will never match. So I can go directly to the next line